metric-anomaly-task-manager/src/main/resources/configs/common/application.conf

```diff
@@ -3,5 +3,40 @@ service.admin.port = 8099
 
 main.class = org.hypertrace.alert.engine.metric.anomaly.task.manager.MetricAnomalyTaskManager
 
+job.config = {
+  cronExpression = "0 * * * * ?"
+  delayInMinutes = 1
+}
+
+ruleSource = {
+  type = "fs"
+  fs = {
+    path = "/app/resources/configs/metric-anomaly-task-manager/rules.json"
+  }
+  dataStore = {
+    dataStoreType = mongo
+    mongo = {
+      host=mongo
+      host=${?MONGO_HOST}
+      port=27017
+    }
+  }
+}
+
+queue.config {
+  type = "kafka"
+  kafka = {
+    topic = "alert-tasks"
+    bootstrap.servers = "localhost:9092"
+    producer = {
```

Comment:
nit: empty blocks

Reply:
Kept it here for example, in case if we want to override any property.